### PR TITLE
Update OpenChain Telco SBOM Specification.md

### PR DESCRIPTION
**Pull request by Jimmy-ahlberg**
Changed Data Format to format where it applies to the format of the SPDX (Tag:Value).

### CODE REVIEW

File: OpenChain Telco SBOM Specification.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This specification is licensed under [Creative Commons Attribution License 4.0 (
 
 ## 2. Terms and definitions
 
-### Data Format
+###
```
Comment (stephenkilbaneadi) [inline OpenChain Telco SBOM Specification.md:15]: Did you mean to remove the title here?

File: OpenChain Telco SBOM Specification.md [hunk 2]
```diff
@@ -79,22 +79,22 @@ NTIA minimum elements https://www.ntia.doc.gov/report/2021/minimum-elements-soft
...
+The Telco Standard SBOM will include, at a minimum, the SPDX in the following machine readable format as default: Tag:Value
 
 #### 3.5.1 Verification and reference material
 Tag:Value is described here in SPDX 2.2 https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements
 
 #### 3.5.2 Rationale
-To facilitate a simplified toolchain, a machine readable version of the SBOM needs to be included. To ensure repeatability and harmonization a conformant SBOM must be in the Tag:Value format. An entity can release additional machine readable Data Formats but they are not required to conform to the Telco SBOM specification.
+To facilitate a simplified toolchain, a machine readable version of the SBOM needs to be included. To ensure repeatability and harmonization a conformant SBOM must be in the Tag:Value format. An entity can release additional machine readable formats but they are not required to conform to the Telco SBOM specification.
 
 ### 3.6 Human Readable Data Format
```
Comment (stephenkilbaneadi) [inline OpenChain Telco SBOM Specification.md:90]: Missed an instance of Data Format in the title.

**Review by stephenkilbaneadi — CHANGES_REQUESTED** [review]
Looks good, but there are a couple of tweaks to make.

File: OpenChain Telco SBOM Specification.md (hunk 1 above)
Comment (Jimmy-ahlberg) [reply OpenChain Telco SBOM Specification.md:15]: I did not, I blame staying up past my bedtime doing work ;)

**Review by vargenau — APPROVED** [review]

**Pull request merged**